lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -242,7 +243,24 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 		const trashSelector = By.css( 'button.editor-post-trash' );
 
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, trashSelector );
-		return await driverHelper.clickWhenClickable( this.driver, trashSelector );
+		await driverHelper.clickWhenClickable( this.driver, trashSelector );
+
+		// wait for 'Move to trash' button to disappear
+		try {
+			return await driverHelper.waitTillNotPresent(
```

Comment:
Wait for "Move to trash" button to disappear before continuing to the next step. Timeout is increased here because of https://github.com/Automattic/wp-calypso/issues/28823